distribution/lib/Standard/Base/0.0.0-dev/src/System/File_Format.enso

```diff
@@ -16,6 +21,18 @@ polyglot java import org.enso.base.file_format.FileFormatSPI
 format_types : Vector
 format_types = Vector.from_polyglot_array (FileFormatSPI.get_types False)
 
+## PRIVATE
+   Gets the first format not returning Nothing from the callback
+get_format : Function -> Any | Nothing
+get_format ~callback =
```

Comment:
```suggestion
## PRIVATE
   Gets the first format not returning Nothing from the callback.
get_format : Function -> Any | Nothing
get_format callback =
```

The `callback` is an _unapplied_ function, so I don't think there is any sense in passing it suspended. It is just a closure anyway, suspending it doesn't really change anything, but may potentially incur additional cost (and may surprise the reader as to why the argument is suspended).